controllers/csiscaleoperator_controller.go

```diff
@@ -665,24 +813,47 @@ func (r *CSIScaleOperatorReconciler) reconcileClusterRole(instance *csiscaleoper
 			Namespace: cr.Namespace,
 		}, found)
 		if err != nil && errors.IsNotFound(err) {
-			logger.Info("creating a new ClusterRole", "Name", cr.GetName())
+			logger.Info("Creating a new clusterRole", "Name", cr.GetName())
```

Comment:
Let's keep the cases for k8s resources consistent. I see different format of cases used for k8s resources. e.g. clusterRole, ClusterRoleBinding. I think it is good to keep it as it is to be in sync with k8s resource `kind` like ClusterRole, ClusterRoleBinding, etc.

Reply:
Fixed. Give it a check once.